pom.xml

```diff
@@ -481,6 +489,14 @@ under the License.
       <plugin>
         <groupId>org.eclipse.sisu</groupId>
         <artifactId>sisu-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>generate-index</id>
+            <goals>
+              <goal>main-index</goal>
+            </goals>
+          </execution>
+        </executions>
```

Comment:
There is a definition in parent pom ....
https://github.com/apache/maven-parent/blob/master/pom.xml#L1014-L1027

with additional we execute plugin twice:

```
[INFO] --- sisu:0.9.0.M3:main-index (index-project) @ maven-javadoc-plugin ---
...
[INFO] --- sisu:0.9.0.M3:main-index (generate-index) @ maven-javadoc-plugin ---
```

Reply:
good catch. removed.